en/console-and-shells.rst

```diff
@@ -648,21 +648,27 @@ The methods that allow chaining are:
 - addSubcommand()
 - addSubcommands()
 - command()
+- setCommand()
 - description()
-- epilog()
+- setDescription()
+- setEpilog()
 
-.. php:method:: description($text = null)
+Set the Description
+~~~~~~~~~~~~~~~~~~~
 
-Gets or sets the description for the option parser. The description displays
-above the argument and option information. By passing in either an array or a
-string, you can set the value of the description. Calling with no arguments will
-return the current value::
+.. php:method:: setDescription($text)
+
+The description displays above the argument and option information. By passing
```

Comment:
Probably: "The description **above** displays the argument and option information."